api/actions/views.py

```diff
@@ -75,10 +82,21 @@ class ActionDetail(JSONAPIBaseView, generics.RetrieveAPIView):
     view_category = 'actions'
     view_name = 'action-detail'
 
+    def get_serializer_class(self):
+        # Not allowed to view NodeRequestActions yet, making extra logic unnecessary
+        # obj = self.get_object()
```

Comment:
Assuming these commented out bits are for later on? Since they're simple enough, may make sense to remove them for now for realz and put them back when it's needed?

Reply:
True. I had them in for real, realized that view permissions for these objects was "none yet", and just commented it out with a brief explanation. Since we'll be moving to `django-guardian` (hopefully) soon, this might be obviated by the time it becomes necessary anyway. I'll remove.